pkg/products/monitoring/dashboards/resourceByNamespace.go

```diff
@@ -423,7 +423,7 @@ const MonitoringGrafanaDBResourceByNSJSON = `{
 			"stack": false,
 			"steppedLine": false,
 			"targets": [{
-				"expr": "sum(container_memory_usage_bytes{namespace=~'$namespace', container!=''}) by (pod)",
+				"expr": "sum(container_memory_working_set_bytes{namespace=~'$namespace', container=''}) by (pod)",
```

Comment:
Changes look good to me, verification worked with the steps above.

The only question I have (Just to get a better understanding) is why do we remove the `!=` operator? my understanding would be that it doesn't actually make a difference in this query, since the previous way just searched for all containers that don't have an empty name  i.e. `!=''` , while with your changes we search for all containers ( `''` being a syntactical equivalent of a non name specific search?) essentially looking for all containers no matter what their label is within this namespace? correct me if I'm wrong, thanks!

/lgtm 
/Approve

Reply:
Good question, So both the expressions with != or just = will get you similar results as they are essentially doing the same thing. The difference between them is with the != your manually adding up all the memory resources needed individually to get the value. Whats removed with the !=  is a result that has all these resources added up already but is also bringing in certain memory resources being used that we are missing when summing the values manually. So using the = rather then != is the value coming back is more accurate as its including certain resources we are not getting.